### ISSUE TITLE
What‘s paramaters the paper used？

### ISSUE THREAD

**Issue by huaileiseu** [Atomu2014/product-nets-distributed#1]
I do some experiments use this repo following the paper, but the auc and loss have a gap between my experiments and the paper, maybe my paramaters are wrong, so can you provide the command lines the paper used. Thanks

**Comment by Atomu2014** [comment]
Hi,

There are too many experiment records. If you can provide more details about your settings, I can help you fix this problem.

**Comment by huaileiseu** [comment]
I use the paramaters got from Table 5 show in paper. The commands for deepfm are following,
nohup python2 -u tf_main.py --distributed=False --num_gpus=1 --dataset=avazu --model=deepfm --batch_size=2000 --eval_level=0 --optimizer=adam --learning_rate=1e-3 --embed_size=40 --l2_embed=0 --l2_kernel=0 --num_rounds=20 --nn_layers="[[\"full\", 500], [\"act\", \"relu\"], [\"full\", 500], [\"act\", \"relu\"], [\"full\", 500], [\"act\", \"relu\"], [\"full\", 500], [\"act\", \"relu\"], [\"full\", 500], [\"act\", \"relu\"], [\"full\", 1]]" > resultDeepFM_avazu.txt 2>&1 & 

the best auc: 0.781810 loss:0.378762

and use ln:
nohup python2 -u tf_main.py --distributed=False --num_gpus=1 --dataset=avazu --model=deepfm --batch_size=2000 --eval_level=0 --optimizer=adam --learning_rate=1e-3 --embed_size=40 --l2_embed=0 --l2_kernel=0 --num_rounds=20 --nn_layers="[[\"full\", 500], [\"ln\", \"\"], [\"act\", \"relu\"], [\"full\", 500], [\"ln\", \"\"], [\"act\", \"relu\"], [\"full\", 500], [\"ln\", \"\"], [\"act\", \"relu\"], [\"full\", 500], [\"ln\", \"\"], [\"act\", \"relu\"], [\"full\", 500], [\"ln\", \"\"], [\"act\", \"relu\"], [\"full\", 1]]" > resultDeepFM_avazu.txt 2>&1 &

the best auc: 0.781523 loss:0.378853

nohup python2 -u tf_main.py --distributed=False --num_gpus=1 --dataset=avazu --model=pin --batch_size=2000 --eval_level=0 --optimizer=adam --learning_rate=1e-3 --embed_size=40 --l2_embed=0 --l2_kernel=0 --num_rounds=20 --nn_layers="[[\"full\", 500], [\"act\", \"relu\"], [\"full\", 500],  [\"act\", \"relu\"], [\"full\", 500],  [\"act\", \"relu\"], [\"full\", 500], [\"act\", \"relu\"], [\"full\", 500], [\"act\", \"relu\"], [\"full\", 1]]" --sub_nn_layers="[[\"full\", 40], [\"ln\", \"\"], [\"act\", \"relu\"], [\"full\", 5],  [\"ln\", \"\"]]" > resultPIN_avazu.txt 2>&1 &

Experiments were repeated three times, the results of them were similiar. Maybe  I didn't make good use of "ln". Thanks.

**Comment by huaileiseu** [comment]
And I have sent an email to you. Thanks for you patience.
